spark/src/main/scala/geotrellis/spark/ingest/Ingest.scala

```diff
@@ -61,15 +61,14 @@ object Ingest {
       cacheLevel: StorageLevel = StorageLevel.NONE,
       resampleMethod: ResampleMethod = NearestNeighbor
     )
-    (sink: (RasterRDD[K], Int) => Unit)
-    (implicit tiler: Tiler[T, K, Tile]): Unit =
+    (sink: (RasterRDD[K], Int) => Unit): Unit =
   {
     sourceTiles.persist()
     val reprojectedTiles =
       sourceTiles.reproject(destCRS, resampleMethod).cache()
     val (zoom, rasterMetaData) =
       RasterMetaData.fromRdd(reprojectedTiles, destCRS, layoutScheme)(_.projectedExtent.extent)
-    val tiledRdd = tiler(reprojectedTiles, rasterMetaData, resampleMethod).cache()
+    val tiledRdd = reprojectedTiles.cutTiles(rasterMetaData, resampleMethod).cache()
```

Comment:
Isn't this missing something that will `.reduceByKey` on the cut tiles?
It looks like this could a call to `tileToLayout`.